pass-nihms-loader/nihms-etl-util/src/main/java/org/eclipse/pass/loader/nihms/util/ProcessingUtil.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2023 Johns Hopkins University
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.eclipse.pass.loader.nihms.util;
+
+import java.time.LocalDate;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Collection;
+
+/**
+ * General small utilities to tidy up data processing code.
+ *
+ * @author Karen Hanson
+ */
+public class ProcessingUtil {
+
+    private ProcessingUtil () {
+        //never called
+    }
+
+    private static final String DEFAULT_DATE_PATTERN = "yyyy-MM-dd";
+
+    /**
+     * Returns true if a string is empty or null
+     *
+     * @param str the string, may be {@code null}
+     * @return true if a string is empty or null
+     */
+    public static boolean nullOrEmpty(String str) {
```

Comment:
Done.

Reply:
@rpoet-jh Actually not sure if this done. Did you want me to replace only the method nullOrEmpty() or all the methods in the ProcessingUtil class? The formatDate() is doing a little more, so it might be harder to replace.

Reply:
Just the string operation methods, so it's done.

Reply:
Discussed with Russ, instead of using isBlank(), will use isEmpty() and use isNotEmpty instead of !isEmpty().